swatch-contracts/src/main/resources/application.properties

```diff
@@ -203,13 +233,15 @@ quarkus.rest-client."com.redhat.swatch.clients.product.api.resources.ProductApi"
 quarkus.rest-client."com.redhat.swatch.clients.product.api.resources.ProductApi".scope=jakarta.enterprise.context.ApplicationScoped
 quarkus.rest-client."com.redhat.swatch.clients.product.api.resources.ProductApi".providers=com.redhat.swatch.resteasy.client.DebugClientLogger
 # Setting up the timeout to 60 seconds instead of 30 seconds (default)
-quarkus.rest-client."com.redhat.swatch.clients.product.api.resources.ProductApi".read-timeout=90000
+quarkus.rest-client."com.redhat.swatch.clients.product.api.resources.ProductApi".read-timeout=60000
```

Comment:
why this change?

Reply:
I did that while debugging.  Now reverted.